docs/developer-tools/runtime/runtime-protocol-reference/marlowebulksync-subprotocol.md

```diff
@@ -0,0 +1,55 @@
+---
+title: MarloweBulkSync sub-protocol
+sidebar_position: 3
+---
+
+The MarloweBulkSync sub-protocol is defined here: 
+
+- **[https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe-runtime/sync-api/Language/Marlowe/Protocol/BulkSync/Types.hs](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe-runtime/sync-api/Language/Marlowe/Protocol/BulkSync/Types.hs)**
+
+The MarloweBulkSync sub-protocol is used to synchronize all marlowe contract transactions. 
+The client receives a stream of blocks, each of which contains collections of create, apply inputs, and withdraw transactions.
+
+This is a firehose protocol. It gives far more information than the header sync
+or regular sync protocols, but for following a small set of contracts may be an
+inefficient choice.
+
+### Sub-protocol states
+
+| Protocol state | Agency | Description |
+| --- | --- | --- |
+| 1. `Idle` | `Client` | The initial state. | 
+| 2. `Intersect` | `Server` | When the client asks the server to fast forward to a known chain point. | 
+| 3. `Next` | `Server` | When the client asks the server to provide the next block of marlowe transactions. | 
+| 4. `Poll` | `Client` | When the server has no more blocks to send (i.e., the client has reached the tip of the chain). | 
+| 5. `Done` | `Nobody` | The terminal state. | 
+
+### The Intersect state 
+
+The Intersect is when the client asks the server to fast forward to a known chain point. 
+For example, in a situation where the client has already done a fair amount of synchronization, and then quits, when it starts up again, rather than starting over from Genesis, it can instead start from the point where it last left off. 
```

Comment:
Suggested minor edit: 

For example, in a situation where the client has already done a fair amount of synchronization and then quits, when it starts up again, rather than starting over from Genesis, it can start from where it last left off.

Reply:
```suggestion
For example, in a situation where the client has already done a fair amount of synchronization and then quits, when it starts up again, rather than starting over from Genesis, it can start from where it last left off.
```